microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

```diff
@@ -13,6 +13,39 @@ This page lists the updates to PWAs and Web Apps from the Microsoft Edge Web App
 
 
 <!-- ====================================================================== -->
+## What's New in Microsoft Edge 100
+Microsoft Edge Canary/Dev channels moved to version 100 on February 9,2022 and in this milestone, we are testing the following new web app features. 
+
+### App Sync
+The sync service powering features like favorites, settings, history & more has been expanded to include your installed web apps - that is PWAs and sites installed as apps. The sync service works when you are signed in to your profile and have opted in to syncing. As we test and gradually release this feature over the coming weeks, users that have opted in to sync will start to see "Available apps" in the `edge://apps` page and the [new Apps menu](#redesigned-apps-menu). 
+
+Available apps will show any apps you've installed on other devices with the same signed in profile, that aren't currently installed in the Edge instance you are using. The available apps will be shown in their own section with grey icons. Press the "Install" button to install the app on that device.
+
+   :::image type="content" source="media/available-app-install.png" alt-text="Available apps shown on Apps page with install button highlighted.":::
+
+App sync is available on desktop devices only at this stage. To keep your apps in sync across devices, adding an app on one device will add it to the "Available apps" section on other devices. Uninstalling an app on one device will remove it from other devices.
+
+### Redesigned Apps Menu
+The Apps menu item within Edge's "Settings and more" menu is being redesigned so that the menu icon can be added to the toolbar for quick access and the popup menu can be pinned open as a sidebar to show your apps alongside web content. Customizable sort and view options have been added along with an "Available apps" section to support the app sync feature described above.
```

Comment:
```suggestion
The **Apps** menu item within Microsoft Edge's **Settings and more** menu is being redesigned so that the menu icon can be added to the toolbar for quick access and the popup menu can be pinned open as a sidebar to show your apps alongside web content. Customizable sort and view options have been added along with an **Your available apps** section to support the app sync feature described above.
```